aspnetcore/mvc/views/razor.md

```diff
@@ -92,6 +100,20 @@ You can use an explicit expression to concatenate text with an expression result
 
 Without the explicit expression, `<p>Age@joe.Age</p>` is treated as an email address, and `<p>Age@joe.Age</p>` is rendered. When written as an explicit expression, `<p>Age33</p>` is rendered.
 
+You can also use explicit expressions to render output from generic methods in _.cshtml_ files.  In an implicit expression, the characters inside the brackets (`<>`) would be interpreted as an HTML tag:
+
+```cshtml
+<p>@GenericMethod<int>()</p>
+```
+
+The code generates a compiler error like `The "int" element was not closed.  All elements must be either self-closing or have a matching end tag` or `Cannot convert method group 'GenericMethod' to non-delegate type 'object'. Did you intend to invoke the method?`  However, when written as an explicit expression, the generic method's output will compile correctly.  
```

Comment:
* Remove one space: "method?`  However"
* For "However, when" ... either drop "However" or drop the comma. You can probably just drop the "However" ... the MS Style manual hints that they prefer not to stack opening prepositional phrases and/or conjunctive adverbs.
* will compile :point_right: compiles

Reply:
👍

Reply:
Oh ... I meant to mark the comment itself ... sorry ... I wasn't clear ...

![capture](https://user-images.githubusercontent.com/1622880/31631548-991466a8-b27f-11e7-9ab6-c5c5baf6e660.PNG)